### PR TITLE
Feature/KG-497

### PR DESCRIPTION
**Pull request by DavidIzaac**

### CODE REVIEW

File: console/src/app/admin/admin.component.scss [hunk 1]
```diff
@@ -36,16 +36,14 @@ ul.nav {
...
         cursor: pointer;
       }
 
     }
 
     &.active {
-      background: $selected-tab-environment;
       a.nav-link {
-        color: $turquoise-blue
+        color: #f79f2b;
```
Comment (kadosh1000) [inline console/src/app/admin/admin.component.scss]: @DavidIzaac move to variable, also check if it is already exists in some variable

File: console/src/app/agents/agents-list/agents-list.component.scss [hunk 2]
```diff
@@ -22,12 +13,57 @@
   }
 
   .row{
-    .col-3{
-      padding-left: 0;
+    .groups{
+      padding-left: 4px;
+      padding-right: 25px;
+      width: 30%;
+      height: 810px;
```
Comment (kadosh1000) [inline console/src/app/agents/agents-list/agents-list.component.scss]: @DavidIzaac height should not be set all across, remove all heights

File: console/src/app/agents/edit-agent/edit-agent.component.html [hunk 3]
```diff
@@ -11,11 +11,23 @@ <h4>Edit agent</h4>
         <label>Name:</label>
         <input type="text" id="title" class="form-control" [(ngModel)]="name"/>
       </div>
-      <tag-input [(ngModel)]="attributes" [editable]='true' theme='minimal'></tag-input>
+      <div class="form-group">
+        <label>Add tag</label>
+        <div class="buttonInside">
+          <input [(ngModel)]="tag" (keyup)="keyUp($event,tag)">
+          <a (click)="saveTag(tag)" ><i [ngClass]="{'isValid': isValid}" class="material-icons">done</i></a>
```
Comment (DavidIzaac) [inline console/src/app/agents/edit-agent/edit-agent.component.html]: replace with hidden

File: server/api/services/agents.service.js [hunk 4]
```diff
@@ -431,14 +445,37 @@ module.exports = {
...
+
     /**
      * Removing an agent from a group
      * @param groupId
      * @param agentId
      * @returns {Query|*}
      */
     removeAgentFromGroup: (groupId, agentId) => {
-        return Group.findOneAndUpdate(groupId, { $pull: { agents: { $in: [agentId] } } }, { new: true });
+        return new Promise((resolve,reject) => {
```
Comment (kadosh1000) [inline server/api/services/agents.service.js]: remove the new Promise, the following line (`Group.findOne`) already returns a promise

**Pull request merged**